### PR TITLE
New documentation S.Port Firmware Flashing

### PR DESCRIPTION
**Pull request by RCdiy**

### CODE REVIEW

File: s-port_flashing.md [hunk 1]
```diff
@@ -1,2 +1,108 @@
...
+
+## File Preparation
+
+- Download the firmware
+
+  ![](images/sport-flash-001-download-firmware.png)
+  
+- Place the firmware on the SDCARD in the FIRMWARE folder
+  
+  ![](images/sport-flash-104-sd-hc-firmware.bmp)
```
Comment (schwabe) [inline s-port_flashing.md]: Convert the bmp to png if possible.

File: s-port_flashing.md [hunk 2]
```diff
@@ -1,2 +1,108 @@
...
+
+  ![](images/sport-flash-106-sd-hc-firmware-file-selected.bmp)
+  
+## Hardware Preparation
+
+### All transmitters except for Taranis Q X7 shipped with bottom S.Port pins
+
+- The external module bay’s +ve pin has the same voltage as the transmitter’s battery
+
+  Check that the device being flashed is rated for the transmitter’s battery voltage
```
Comment (schwabe) [inline s-port_flashing.md:23]: Add a paragraph: This is true for X9D+ standard batteries (6 cell Nimah) and most Sport devices . For X12S and X9E this is not true but only slightly outside the range and reportely works. But try at your own risk.

File: s-port_flashing.md [hunk 3]
```diff
@@ -1,2 +1,108 @@
...
+- Use a regular servo cable
+
+- Plug one end into the bottom S.Port
+
+- Plug the other end onto the S.Port pins of the device to be flashed
+
+### Taranis Q X7 bottom S.Port pins added
+
+- Will not work
+
```
Comment (schwabe) [inline s-port_flashing.md:50]: Remove these two lines and add a something like

Note: Adding manually the Sport pins to a X7 variant shipped without the sport pins will not work as this requires also a newer PCB revision.

to the non sport section.

**Review by schwabe — CHANGES_REQUESTED** [review]

File: s-port_flashing.md (hunk 1 above)
Comment (RCdiy) [reply s-port_flashing.md]: How do I do this? Update the file in my fork and do a new pull request? Not sure how this works...

File: s-port_flashing.md (hunk 3 above)
Comment (RCdiy) [reply s-port_flashing.md:50]: Same as saying will not work :)

File: s-port_flashing.md (hunk 1 above)
Comment (RCdiy) [reply s-port_flashing.md]: If bmp works lets leave it... I'll change them later or someone else can:)

File: s-port_flashing.md (hunk 2 above)
Comment (RCdiy) [reply s-port_flashing.md:23]: Making it more verbose unnecessarily... I'm focusing on the flashing...
But... the extra wording can be added by anyone once posted:)

**Comment by RCdiy** [comment]
Future documentation will be posted in an issue. It can be edited there and once consensus reached the file will be updated and a pull request made.
My thoughts are to accept this pull request and either an admin makes the changes or waits till I do:)
Mike Barsalou and I are working on a draft for a workflow wiki so we are all on the same page:)

**Comment by RCdiy** [comment]
BMP changed to PNG.
Please merge.

**Pull request closed without merging**